js/app.js

```diff
@@ -0,0 +1,131 @@
+// HTML tag variables
+let person4Name = document.querySelector("#person4Name");
+let person4HomeWorld = document.querySelector("#person4HomeWorld");
+let person14Name = document.querySelector("#person14Name");
+let person14Species = document.querySelector("#person14Species");
+let filmList = document.querySelector("#filmList");
+
+// Create XHR for Vader
+const person4XHR = new XMLHttpRequest();
+person4XHR.addEventListener("load", reqName1);
+person4XHR.open("GET", "https://swapi.co/api/people/4/", true);
+person4XHR.send();
+
+// Pull name from API and append to person4Name div
+function reqName1() {
```

Comment:
Choose better variable names for your functions. reqSpecies and showFilms is fine.